php_phongo.h

```diff
@@ -25,9 +25,8 @@
 extern zend_module_entry mongodb_module_entry;
 
 /* FIXME: Its annoying to bump version. Move into phongo_version.h.in */
-#define MONGODB_VERSION_S "1.2.8-dev"
-#define MONGODB_STABILITY_S "devel"
-#define MONGODB_VERSION   MONGODB_VERSION_S
+#define PHP_MONGODB_VERSION "1.2.8-dev"
+#define PHP_MONGODB_STABILITY "devel"
```

Comment:
Arguments of the [extension skeleton](https://github.com/php/php-src/blob/php-7.1.3/ext/skeleton/php_skeleton.h#L9) aside, these names are also more consistent with the `MONGODB_VERSION` and `MONGODB_STABILITY` constants we define for userland.

As for `MONGODB_VERSION`, I found no references to it apart from `zend_module_entry`. This seemed ripe for removal after updated the struct to use `PHP_MONGODB_VERSION`.